src/workloads/transactions/LLTAnalytics.yml

```diff
@@ -0,0 +1,284 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Workload to Benchmark the effect of LongLivedTransactions on an Update workload.
+
+GlobalDefaults:
+  # Start: The following block gets updated for the other configurations.
+  # Not In-memory: Database size works out about 30GB.
+  InitialDocumentCount: &InitialNumDocs 4900000 #0
```

Comment:
is there also a LTC version?

Reply:
The comment remained from your script - will need to be adjusted. The current size is 21.9GB for /data/dbs after the test. It appears to be LTC - both xvde and xvdf have zero reads (which is somewhat strange). Marketing wants drastically larger set of data - 80M documents according to the ticket - and we have 5.9M here.